src/ENDFtk/resonanceParameters/resolved/RMatrixLimited/src/AP.hpp

```
@@ -0,0 +1,20 @@
+/**
+ *  @brief Return the scattering radius
+ *
+ *  This function will only return the first true scattering radius value it
+ *  finds in the first spin group. If other spin groups use different radii,
+ *  AP is meaningless and should not be used as a shortcut to retrieve
+ *  the scattering radius.
```

Is a note saying "you shouldn't do this" a strong enough check?  Is it feasible to throw an exception if AP is called in a case where it's meaningless?

---

True, but I don't  want to throw an exception at this point. LRF7 is not widely used and I think most evaluations use the same AP.